### DISCUSSION TITLE
How do you use TRPC with Prisma JSON Value?

### DISCUSSION THREAD

**Discussion by mxvsh** [trpc/trpc#5511] [Q&A]
I am returning Prisma query from a TRPC call, and it has JSON value in it, on the client side, I am always getting TypeScript error :

`Type instantiation is excessively deep and possibly infinite.ts(2589)`

**Comment by KATT** [comment]
Create an issue with a reproduction 🙏 sounds like our ts json serialize and their json type doesn't play nicely together
